docs2/docs/upgrade-guides.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: upgrade-guides
+title: Upgrade guides
+joern_shared: true
+---
+
+
+## 1.1.1: OverflowDb Traversals
+This release introduces a major rearchitecture of the cpg query language (CPGQL). Most changes happened under the hood, however there are a few changes that are user facing. The migration should be straightforward, and in any case we're here to help.
+
+Background: CPGQL was previously based on the [Gremlin graph traversal language](https://tinkerpop.apache.org/gremlin.html) (accessible via `.raw`), and is now based on [OverflowDb Traversal](https://github.com/ShiftLeftSecurity/overflowdb/blob/master/traversal/src/main/scala/overflowdb/traversal/Traversal.scala). The main drivers behind this change are: better performance, less complexity and fewer dependencies. OverflowDb Traversal is a Scala collection with additional graph steps, which means we now inherit many useful steps from the rich Scala standard collection library, where previously convertions between Traversal and Scala collections were needed.
+At the same time, Traversal offers largely the same steps and semantics as its TinkerPop predecessor. 
+
+Here are the most important new parts - from our experience the swap of `filter` and `where` accounts for 90% of user-facing changes:
+1) `filter(A => Boolean)`: just like any other Scala collection, filter now simply takes a predicate (it used to take a Traversal). 
+2) `where(trav: Traversal[A] => Traversal[_])`: preserves elements if the provided traversal has at least one result. This is what filter used to be. 
```

Comment:
```suggestion
2. `where(trav: Traversal[A] => Traversal[_])`: preserves elements if the provided traversal has at least one result. This is what filter used to be. *Effectively, `filter` and `where` are swapped*. 

```

A newline after a list item is a Markdown convention that some parsers prefer, I think it would be cool to keep it.
Same with `1)` vs `1.` the latter is more common, and some parsers interpret the previous one differently, it would be nice to stay consistent and use the dot format `1.`